modules/sbo-configuration-of-directory-path-to-project-binding-data.adoc

```diff
@@ -0,0 +1,68 @@
+[id="sbo-configuration-of-directory-path-to-project-binding-data_{context}"]
+= Configuration of the directory path to project the binding data inside workload container
+
+For projecting the binding data, by default, files are mounted at a specific directory in your workload resource. You can configure the directory path using the existing `SERVICE_BINDING_ROOT` environment variable setup in the container where your workload runs.
```

Comment:
Consider rephrasing this based on the suggestion for the previous file.

Reply:
Still,  in the first sentence, please avoid the passive voice by naming the thing that mounts the files. 

Often, when a user encounters a passive phrase like this one, they have to carry a sort of mental burden until they come across the answer to the mystery. Too many of these burdens and they will end up tired and frustrated.